bin/osg-authfile

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3.6
+# -*- coding: utf-8 -*-
+
+"""
+A script to generate Authfiles for XCaches in the OSG
+"""
+
+import webapp.models
+import stashcache
+
+global_data = webapp.models.GlobalData()
```

Comment:
This needs to be configurable -- I certainly don't have a "contacts" checkout inside my topology dir. Since you probably don't want to make your own config system, I suggest you use Flask's, like the webapp does -- see lines 38-58 of `src/app.py`.

Reply:
Right now, I see `osg-authfile` as a testing tool for developers (since the primary use will be via the web interface).  Hence, I don't worry too much about providing a separate config file for this.

I also don't have a `contacts` directory checked out -- it warns about it and moves on.

Reply:
In that case, do you think it's worth adding `osg-authfile` to the CI?

Reply:
Yes!  What's the best way to do that?

Reply:
For a minimal test (run it and see if it crashes), just add it to `.travis.yml`. If need to make a wrapper script, put it in `src/tests`.

Reply:
Done in the latest commit.